pypit/calibrations.py

```diff
@@ -82,6 +102,11 @@ def __init__(self, fitstbl, spectro_class=None, save_masters=True, write_qa=True
         # Internals
         self._reset_internals()
 
+    @staticmethod
```

Comment:
I was intentionally leaving this list empty for the Parent class.

Sort of to force us to use a Child.

Reply:
This function is actually only ever called by the child at the moment.  I saw this as the default function that a child could overwrite if it needed.  I can move it back to the child, if that makes more sense.